controllers/argocd/applicationset/constants.go

```diff
@@ -0,0 +1,9 @@
+package applicationset
+
+// Values
+const (
+	ArgoCDApplicationSetControllerComponent = "applicationset-controller"
+	ArgoCDApplicationSetController          = "argocd-applicationset-controller"
+	ApplicationSetGitlabSCMTlsCert          = "appset-gitlab-scm-tls-cert"
+	ApplicationSetGitlabSCMTlsCertPath      = "/app/tls/scm/cert"
```

Comment:
can we make this uniform? 
`AppSetControllerComponent`, `AppSetGitlabSCMTlsCert` etc

Reply:
In `notifications` controller, they are 
```
	ArgoCDNotificationsControllerComponent = "notifications-controller"
	NotificationsSecretName                = "argocd-notifications-secret"
	NotificationsConfigMapName             = "argocd-notifications-cm"
```
So I followed the same convention.

Do you want them changed to the below names?
```
NotificationsControllerComponent
NotificationsSecretName
NotificationsConfigMapName

AppSetControllerComponent
AppSetController
AppSetGitlabSCMTlsCert
AppSetGitlabSCMTlsCertPath
```

Reply:
yeah, I dont think we need ArgoCD in every constant 
we can reserve that for the general constants, and component specific constants can just start with the component name